models/nxos/lldp/merged_example_01.txt

```diff
@@ -0,0 +1,19 @@
+# Using merged
+# Before state:
+# -------------
+#
```

Comment:
It's probably a good practice to specify the show command that fetches lldp config and its output, even if its empty. You can refer: https://github.com/ansible/network/pull/40/files#diff-1bb42e3752caa463e06497b7db60b669R84

Note: `% No such configuration item(s)` is what IOS-XR shows when no config items are found for a show command. NX-OS maybe different.